11-payment-encoding.md

```diff
@@ -141,6 +141,7 @@ Currently defined tagged fields are:
    * `fee_base_msat` (32 bits, big-endian)
    * `fee_proportional_millionths` (32 bits, big-endian)
    * `cltv_expiry_delta` (16 bits, big-endian)
+* `v` (12): `data_length` variable.  Optional name of vendor/supplier (UTF-8).
```

Comment:
The tag could be used for both B2C and P2P payments, so it makes sense to keep this generic.

```suggestion
* `v` (12): `data_length` variable.  Optional name of recipient (UTF-8).
```

(The `r` tag is already used for route hints.)